backend/index.js

```diff
@@ -0,0 +1,28 @@
+const { Prisma } = require('prisma-binding');
+const { ApolloServer } = require('apollo-server');
+const { importSchema } = require('graphql-import');
+
+const typeDefs = importSchema('backend/src/schema.graphql');
+const Query = require('./src/Query');
+const Mutation = require('./src/Mutation');
+
+const db = new Prisma({
+  typeDefs: 'backend/generated/prisma.graphql',
```

Comment:
For `index.js` root folder is `backend` so you should define path as `typeDefs: 'generated/prisma.graphql'`